tools/CreateFilterMappings.ps1

```diff
@@ -390,7 +389,7 @@ function Add-TestDllMappings
             }
         }
 
-        $TestProjects = Get-ChildItem -Path $ServiceFolderPath -Filter "*.Test*csproj" -Recurse | where { $_.FullName -notlike "*Netcore*" -and $_.FullName -notlike "*Stack*" }
+        $TestProjects = Get-ChildItem -Path $ServiceFolderPath -Filter "*.Test*csproj" -Recurse | where { $_.FullName -notlike "*Netcore*" -and $_.FullName -notlike "*Stack*" -and $_.FullName -notlike "*Commands.Common.Test*" }
```

Comment:
We should be able to remove this addiotnal condition after this is merged, right?  Because the path won't exist

Reply:
(this is just a question - I think we probably have to have this until this PR is merged).

Reply:
This PR does not remove common code out of azure-powershell. That would be a separate PR.